UnityProject/Assets/Scripts/NPC/AI/Hostile/GenericHostileAI.cs

```diff
@@ -23,23 +24,18 @@ namespace Systems.MobAIs
 	[RequireComponent(typeof(ConeOfSight))]
 	public class GenericHostileAI : MobAI
```

Comment:
![](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ffandomwire.com%2Fwp-content%2Fuploads%2F2022%2F08%2Fgiancarlo-esposito.webp&f=1&nofb=1&ipt=3debf5e0999243b5ec4c880f1a5bf59cb18ecea7fe3ada1226d892b297c21f20&ipo=images)

Reply:
is slow so optimising

Reply:
you should optimize this by porting the mobs to MobV2
stop touching obsolete code that is going to be nuked sooner or later

Reply:
well if it's slowing down the server it slowing down the server is nothing I can do about that